spec/firebase_app_distribution_action_spec.rb

```diff
@@ -1,141 +1,131 @@
 describe Fastlane::Actions::FirebaseAppDistributionAction do
   let(:fake_file) { StringIO.new }
+  let(:fake_connection) { double("Connection") }
+  let(:fake_binary) { double("Binary") }
 
-  before do
-    allow(Tempfile).to receive(:new).and_return(fake_file)
-    allow(fake_file).to receive(:unlink)
-    allow(fake_file).to receive(:path).and_return("/tmp/string")
+  before(:each) do
```

Comment:
I had some thoughts about mocking in tests in this PR https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/pull/61#pullrequestreview-440040774

I think the advice there dovetails nicely with what you've got here, but your specs all in all look very good and you guys can opt to completely disregard if you'd like.